src/index.njk

```diff
@@ -93,7 +93,7 @@ templateClass: home
     <ul role="list">
         {%- for member in team -%}
             <li role="listitem">
-                <img src="{{ member.imageUrl }}" alt="">
+                <img src="{{ member.imageUrl }}" alt="Profile picture of {{member.name}}">
```

Comment:
I'm not against this, but I'm not sure about it.

I set the alt to `""` here because the best description I could think of is just the name that is displayed underneath - Unless we describe the images specially it doesn't add anything, I think 🤔 

Does "Profile Picture" make sense out of context? the pictures were hand-picked so far, I think it's more by chance that we all have the GitHub avatars?

Reply:
How about "Avatar of {{member.name}}"?
I think giving some kind of context about the image is still better then not providing anything

Reply:
Sounds good to me!